arch/risc-v/src/common/espressif/esp_start.c

```diff
@@ -470,15 +478,6 @@ void __esp_start(void)
 
   showprogress('A');
 
-  /* Clear .bss. We'll do this inline (vs. calling memset) just to be
-   * certain that there are no issues with the state of global variables.
-   */
-
-  for (uint32_t *dest = (uint32_t *)_sbss; dest < (uint32_t *)_ebss; )
```

Comment:
So bss is not cleared elsewhere?

Reply:
Yes, it's either cleared by [`bootloader_init`](https://github.com/apache/incubator-nuttx/blob/3446f2a977944813b52014b28143341081dbf6e3/arch/risc-v/src/common/espressif/esp_start.c#L403) or by [`bootloader_clear_bss_section`](https://github.com/apache/incubator-nuttx/blob/3446f2a977944813b52014b28143341081dbf6e3/arch/risc-v/src/common/espressif/esp_start.c#L409)